sdk/python/kfp/_client.py

```diff
@@ -108,6 +108,7 @@ class Client(object):
     cookies: CookieJar object containing cookies that will be passed to the pipelines API.
     proxy: HTTP or HTTPS proxy server
     ssl_ca_cert: Cert for proxy
+    context: Context within kubeconfig to use, defaults to current-context.
```

Comment:
Maybe call it `kubernetes_context` to make it easier to understand?
What is the type of this parameter, BTW?

Reply:
Makes sense - its a string, will add to the docstring and update the name